packages/block-editor/src/components/editor-styles/index.js

```diff
@@ -68,7 +69,21 @@ function useDarkThemeBodyClassName( styles ) {
 
 export default function EditorStyles( { styles } ) {
 	const transformedStyles = useMemo(
-		() => transformStyles( styles, EDITOR_STYLES_SELECTOR ),
+		() =>
+			transformStyles(
+				// Assume that non-SVG styles are CSS that can be transformed.
+				styles.filter( ( style ) => style.__unstableType !== 'svgs' ),
+				EDITOR_STYLES_SELECTOR
+			),
+		[ styles ]
+	);
```

Comment:
This is a potentially subtle issue, but the filter here appears to be to ensure that the `svg` elements provided by the backend data aren't included in `transformedStyles`.

When this change is eventually backported, it'll likely happen at slightly different times, typically with the PHP changes landing before the JS changes.

Would it be possible or worthwhile to have the `svg` elements in a different part of the block editor settings, so that if the backend changes land first, the frontend doesn't get tripped up if this code hasn't landed in the JS side yet?

Reply:
> We might just have to leave a note about it.

That sounds reasonable to me, if it's the right path forward for the feature. It'd be good to make sure the comments are clear in the PHP code for when it comes time to backport, so that we don't forget 🙂

Reply:
I'm not sure if this is what you're talking about, but I added a comment in the php in a0be2bf.

```
// The 'svgs' type is new in 6.3 and requires the corresponding JS changes in the EditorStyles component to work.
```